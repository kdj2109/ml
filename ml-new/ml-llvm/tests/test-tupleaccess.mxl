int main() { 
	int[5] x; 
	int y;
	int z; 
	int a; 
	int b; 
	int c; 
	int d;

	x = [50, 750, 1, 44, 33]; 


	z = x[1];

	print(z);

	d = x[0];
	a = x[2]; 
	b = x[3];
	c = x[4]; 
	
	y = 1000;
	x[1] = y;
	z = x[1];
	print(z);

	print(d);
	print(a); 
	print(b);
	print(c);

	return 0; 
}