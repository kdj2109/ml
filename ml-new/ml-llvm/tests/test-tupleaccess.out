750
1000
50
1
44
33
